docs/contributing/release-checklist.rst

```diff
@@ -47,7 +47,7 @@ release are integrated on ``next`` and all tests are passing. Then:
   - Setup a PR of ``next`` onto ``main``
   - Review the PR
   - Wait for tests to finalize / pass
-  - Double-check the generated docs at https://xnvme.io/docs/next
+  - Double-check the generated docs at https://xnvme.io/en/next
```

Comment:
This confuses me, are we looking to have multiple languages?

Reply:
The previous usage of the ``docs`` folder were fitting when the generated sphinx-doc only lived inside of ``docs``. After changing the sphinx-doc then the folder is for different version of the entire site.Thus, switching to the commonly used convention for multi-version sphinx-docs / read-the-docs.

Reply:
Do we need a prefix? Couldn't we just have `xnvme.io/next`?

Reply:
The convention is to have the language in there.

Reply:
Having said that, then it would be possible but require a bit more work in the ``dest.py`` script to manage this and then of course potential conflicts between branch-names and top-level section names. So, for simplicity I would like to just follow the ``/en/<branch,tag>`` convention.